app/src/main/java/com/ucsd/cse110t15m260/roommateinventory/MainActivity.java

```diff
@@ -41,6 +52,60 @@ protected void onCreate(Bundle savedInstanceState) {
         // ATTENTION: This was auto-generated to implement the App Indexing API.
         // See https://g.co/AppIndexing/AndroidStudio for more information.
         client = new GoogleApiClient.Builder(this).addApi(AppIndex.API).build();
+
+
+        ParseUser.getCurrentUser().logOut();
+        if(ParseUser.getCurrentUser() == null){
+            Person.loginPerson("leo@leo.com","leowong",new LogInCallback() {
```

Comment:
I think @Saiteja64 was just logging in a user as a default for testing/presentation.